src/v2/guide/components-edge-cases.md

```diff
@@ -331,6 +331,8 @@ When the `inline-template` special attribute is present on a child component, th
 </my-component>
 ```
 
+<p>Your inline template needs to be defined inside the DOM element to which Vue is attached.</p>
```

Comment:
The `<p>` tags here are not necessary because it's markdown. 🙂

Reply:
Just created a PR for this https://github.com/vuejs/vuejs.org/pull/2010.
Thanks

Reply:
Great. Much appreciated.